src/app/client/src/app/modules/workspace/components/draft/draft.component.spec.ts

```diff
@@ -19,13 +19,15 @@ import * as mockData from './draft.component.spec.data';
 import { TelemetryModule } from '@sunbird/telemetry';
 import { NgInviewModule } from 'angular-inport';
 import { CoreModule } from '@sunbird/core';
+import { CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
```

Comment:
This is because we are a component named with - in inside this component, without this it is throwing component is not  a known element.
https://angular.io/api/core/CUSTOM_ELEMENTS_SCHEMA

Reply:
Include those components in the mocking module

Reply:
Even if i add that component also it is throwing same error